Dockerfile-cli

```diff
@@ -1,16 +1,43 @@
-FROM docker.io/library/python:3.11-slim-bullseye
+FROM registry.access.redhat.com/ubi8/ubi:latest
 
-RUN addgroup --system nonroot \
-    && adduser --system --ingroup nonroot nonroot
+ENV PYTHON_VERSION=3.11
+
+RUN yum -y update && \
+    yum -y install gcc openssl-devel bzip2-devel libffi-devel zlib-devel make wget && \
+    yum -y install gnupg2 pigz tar && \
+    yum clean all
+
+
+RUN mkdir -p /home/bin
+
+# CG expects these binaries to be availbable in /home/bin
+RUN ln -s /usr/bin/gpg /home/bin/gpg && \
+    ln -s /usr/bin/pigz /home/bin/pigz && \
+    ln -s /usr/bin/tar /home/bin/tar
```

Comment:
I'm quite sure these are not needed on Kasta. But maybe this is intended to run on Hasta as well?

Reply:
Yeah, this is for Hasta